docs/maps/vector-style.asciidoc

```diff
@@ -25,12 +25,16 @@ Use data driven styling to symbolize features from a range of numeric property v
 To enable data driven styling, click image:maps/images/gs_link_icon.png[] next to the property.
```

Comment:
Technically, it should be data-driven styling.  But we should make this change when we update the Ul.